stores/moderationFormInputStore.ts

```diff
@@ -0,0 +1,254 @@
+import { defineStore } from 'pinia'
+import { Ref, ref } from 'vue'
+
+
+export const useModerationFormInputStore = defineStore('moderationFormInputStore', () => {
+    const inputFieldsFacilityForm = ref({
+        contactFields: {
+            nameEn: {
+                label: 'Name En',
+                placeholder: 'e.g. Tachikawa Central Hospital',
+                type: 'text',
+                required: true,
+                inputValid: true,
+                inputValidationMessage: ''
+            },
+            nameJp: {
+                label: 'Name Jp',
+                placeholder: 'e.g. 立川中央病院',
+                type: 'text',
+                required: true,
+                inputValid: true,
+                inputValidationMessage: ''
+            },
+            phone: {
+                label: 'Phone',
+                placeholder: 'e.g. 08080901022',
```

Comment:
I would change the placeholder to something that (probably) isn't a real phone number.

Reply:
Like this ?: 08XXXXXXXXXX